core/src/main/scala/org/http4s/CharsetRange.scala

```diff
@@ -7,21 +7,39 @@ import scalaz.{Equal, Show}
 sealed abstract class CharsetRange extends HasQValue with Renderable {
   def qValue: QValue
   def withQValue(q: QValue): CharsetRange
+
+  @deprecated("Use `isSatisfiedBy` on the `Accept-Charset` header", "0.16.1")
   def isSatisfiedBy(charset: Charset): Boolean
+
+  /** True if this charset range matches the charset.
+    * 
+    * @since 0.16.1
+    */
+  def matches(charset: Charset): Boolean
```

Comment:
I'd accept an argument this shouldn't be made public until 0.18, but our compatibility historically runs in just one direction.